README.md

```diff
@@ -1,20 +1,34 @@
 # Code + Learn
 
-A series of workshop sprints for Node.js.
+Code + Learn is an in-person (and free) event series where developers receive mentoring from Node.js Core contributors to learn the step-by-step process of contributing to Node.js Core.
 
 ## Purpose
 
 Code & Learn events allow you to get started (or go further) with Node.js core
 contributions. Experienced contributors help guide you through your first (or
 second or third or fourth) commit to Node.js core. They also are available to
 provide impromptu guided tours through specific areas of Node.js core source
-code. 
+code.
 
-## Implementation
+## Getting started
 
-*[This is still actively under discussion.](https://github.com/nodejs/code-and-learn/issues/1)*
+- If you don't already have one, [create a GitHub account](https://github.com/join).
+- Open the [Node.js core repository web page](https://github.com/nodejs/node).
+- Use the "Fork" button to fork the repository:![Image](http://nodetodo.org/getting-started/zfork.png)
+- On the page _for your own fork_, use the "Clone or download" button and copy the URL for cloning to your clipboard:![Image](http://nodetodo.org/getting-started/zclone.png)
```

Comment:
Maybe `[Screenshot]` would be a bit more accessible as an alt-text for the images